src/style.css

```diff
@@ -0,0 +1,117 @@
+@font-face {
+  font-family: "Avenir";
```

Comment:
```suggestion
  font-family: "Avenir", sans-serif;
```

Reply:
When I added ,sans-serif in 2 and 8 line, all test was failed

Reply:
bad 🥲
but you need the font family in the end, so don't care too much about test, if you are doing right 🙃